README.md

```diff
@@ -111,3 +111,7 @@ To stop it again:
 
 The docker image at https://quay.io/repository/coredump/status will be
 automatically rebuilt on every push to master.
```

Comment:
Isn't this outdated information after this change?

Reply:
True!

Reply:
Fixed in https://github.com/coredump-ch/status/pull/122.